.github/workflows/build.yml

```diff
@@ -375,23 +205,24 @@ jobs:
           path: internal/command/e2etest/build/terraform-e2etest_${{ matrix.goos }}_${{ matrix.goarch }}.zip
           if-no-files-found: error
 
-  e2etest-linux:
-    name: e2etest for linux_${{ matrix.goarch }}
-    runs-on: ubuntu-latest
+  e2etest:
```

Comment:
@terraform can you confirm that these are the correct e2e tests to run? In the `terraform-releases` Circle CI pipeline, the e2e tests are ran by downloading and executing a `terraform-exec` binary. Do we need to include that here?